packages/components/src/components/contract-card/contract-card-items/toggle-card-dialog.tsx

```diff
@@ -35,8 +36,9 @@ const ToggleCardDialog = ({
     contract_id,
     getCardLabels,
     getContractById,
-    is_valid_to_cancel,
-    should_show_cancellation_warning,
+    is_adding_disabled,
```

Comment:
Renamed: `is_valid_to_cancel` ---> `is_adding_disabled` and `should_show_cancellation_warning` ---> `should_show_warning`